lib/index.ts

```diff
@@ -21,7 +21,7 @@ export {
 };
 
 async function buildDepTree(
-  manifestFileContents: string, lockFileContents: string,
+  manifestFileContents: string, manifestFileName: string, lockFileContents: string,
```

Comment:
Let's maybe make it last argument and make it optional? This way you'll keep API consistent.